index.js

```diff
@@ -52,15 +52,15 @@ controller.hears('invite', 'direct_mention', (bot, message) => {
 /**
  * Private Onboarding
  */
-controller.on('team_join', onboard);
+controller.on('team_join', () => setTimeout(onboard, 5000));
```

Comment:
Yes, can you restore this line and we'll do that one on a separate PR pls so we can merge the other stuff?